docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md

```diff
@@ -16,7 +16,7 @@ description: "Last modified: March 09, 2015"
   
 ## Element information
 
-|||
+| Element Type | Namespace |
```

Comment:
Let's change the headers for this table to "Information/Location" or "Name/Value."